scripts/class_room.py

```diff
@@ -2,4 +2,37 @@
 
 
 class Room(Dojo):
-    pass
+    def __init__(self, room_type, room_name):
+        self.room_type = room_type
+        self.room_name = room_name
+        self.room_capacity = None
+        self.all_rooms = ['White', 'Blue', 'Violet', 'Gray']
+
+    def get_room_capacity(self):
+        if self.room_type.upper() != 'OFFICE':
+            self.room_capacity = 4
+        else:
+            self.room_capacity = 6
+        return self.room_capacity
+
+    def add_new_room(self):
```

Comment:
This should be outside this class. You cannot use a room's instance (class) to create another room.